tool/tctl/common/token_command.go

```diff
@@ -375,3 +382,13 @@ func (c *TokensCommand) List(ctx context.Context, client auth.ClientI) error {
 	}
 	return nil
 }
+
+// template for message when discovery token is being added
+var discoveryMessageTemplate = template.Must(template.New("discovery").Parse(`The invite token: {{.token}}
+This token will expire in {{.minutes}} minutes.
+
+This token enables the Discovery service.  See https://goteleport.com/docs/
```

Comment:
One thing that I dislike about our docs is the fact that everything is condensed inside the Kube, server, db categories...
It's difficult to understand what we really support and point to a specific docs page that references all the options a certain secondary service supports.

Maybe we should consider adding hidden docs pages that appear on search and aggregate all the implementations that a secondary service/feature supports.